crates/solidity/testing/snapshots/cst_output/Statement/try_catch/generated/0.4.11-failure.yml

```diff
@@ -5,13 +5,20 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected AddressKeyword or AsciiStringLiteral or AssemblyKeyword or BoolKeyword or BreakKeyword or ByteKeyword or ContinueKeyword or DecimalLiteral or DeleteKeyword or DoKeyword or FalseKeyword or FixedBytesType or ForKeyword or FunctionKeyword or HexLiteral or HexStringLiteral or Identifier or IfKeyword or MappingKeyword or NewKeyword or OpenBrace or OpenBracket or OpenParen or PayableKeyword or ReturnKeyword or RevertKeyword or SignedFixedType or SignedIntegerType or StringKeyword or ThrowKeyword or TrueKeyword or UnsignedFixedType or UnsignedIntegerType or VarKeyword or WhileKeyword.
-       ╭─[crates/solidity/testing/snapshots/cst_output/Statement/try_catch/input.sol:1:1]
+    Error: Expected Equal or Semicolon.
+       ╭─[crates/solidity/testing/snapshots/cst_output/Statement/try_catch/input.sol:1:6]
        │
      1 │ try a.b() {} catch {}
-       │ ──────────┬──────────  
-       │           ╰──────────── Error occurred here.
+       │      ────────┬───────  
+       │              ╰───────── Error occurred here.
     ───╯
 
 Tree:
-  - SKIPPED (Token): "try a.b() {} catch {}" # 0..21
+  - Statement (Rule): # 0..21 "try a.b() {} catch {}"
+      - VariableDeclarationStatement (Rule): # 0..5 "try a"
+          - VariableDeclaration (Rule): # 0..5 "try a"
+              - TypeName (Rule): # 0..3 "try"
+                  - IdentifierPath (Rule): # 0..3 "try"
+                      - Identifier (Token): "try" # 0..3
```

Comment:
same here. `try` is not a valid identifier in any version.

Reply:
Fixed

Reply:
Is it? I still see `Identifier (Token): "try" # 0..3`

Reply:
`try` as a keyword was introduced in `0.6.0`

Reply:
Thanks for confirming.